mod_example/jni/mcpelauncher_api.h

```diff
@@ -4,8 +4,17 @@
 extern "C" {
 #endif
 
-void* mcpelauncher_hook(void* symbol, void* hook, void** original);
-void mcpelauncher_unhook(void* hook);
+enum class LogLevel {
+    TRACE,
```

Comment:
enum class works only in C++. We want C compatibility, so this has to be made a normal enum and prefixed with LOG_ so we do not pollute the global namespace too much.